package.json

```diff
@@ -144,7 +144,7 @@
     "node-cljfmt": "^0.5.3-1",
     "node-uuid": "^1.4.3",
     "open": "0.0.5",
-    "prettydiff": "^1.16.27",
+    "prettydiff": "git+https://github.com/prettydiff/prettydiff.git#master",
```

Comment:
I will have to reject this for reasons described in https://github.com/Glavin001/atom-beautify/pull/1466#issuecomment-272496867

- [ ] Remove direct Git repo

/cc @prettydiff